django/cantusdb_project/main_app/templates/chant_create.html

```diff
@@ -152,6 +152,13 @@ <h3>Create Chant</h3>
                         {{ form.melody_id }}
                     </div>
 
+                    <div class="form-row align-items-top"> <!-- the Addendum field is no longer there in OldCantus... -->
```

Comment:
if it's not in OldCantus, let's just get rid of it.

Reply:
It is there in OldCantus when you create chants....<img width="592" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/db25614f-e96c-4de2-91b3-62959e1eb675">
It's also in edit 
<img width="660" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/d68c2ae4-0d4d-43f6-9e81-c98ca66f1003">
but not in edit-volpiano 
<img width="655" alt="image" src="https://github.com/DDMAL/CantusDB/assets/67451875/3ef43ffb-60b3-44f7-a898-80d71b780e1d">

Reply:
I realize that we have another discrepancy between OldCantus and NewCantus - OC's "edit-volpiano" is NC's "edit-chants": http://206.12.88.113/edit-chants/699578

I think I may have suggested/made this change last summer. I think "edit-volpiano" doesn't describe what happens on the page nearly as well as "edit-chants", but if we want perfect backwards compatibility, we should change this back.